conf/defaults.ini

```diff
@@ -2071,6 +2071,21 @@ dev_mode = false
 # How much artificial delay to add to store operations. Simulates async k8s behavior. 
 dev_stub_latency = 0
 
+# Used for signing
+secret_key = SW2YcwTIb9zpOOhoPsMm
```

Comment:
I assume this is here on purpose

Reply:
Yes, this is a copy paste of the values under the `[security]` section that we are using for envelope encryption, but I like the idea of relying on our own separate config.